src/Common/Packet.cpp

```diff
@@ -2,6 +2,7 @@
 // Created by PinkySmile on 01/10/2022.
 //
 
+#include <cstdint>
```

Comment:
I'm assuming you had compiler error in these files?

Reply:
no error hear iirc. probably editor added them automatically when i used uint16_t in this file